### DISCUSSION TITLE
Scene works as intended but I get a "case mismatch" message

### DISCUSSION THREAD

**Discussion by WyvernZ22** [ramokz/phantom-camera#300] [Question / Help Wanted] [answered]
I updated the phantom camera to the new version but had some issues, so I had to uninstall and install the plugin gain.  The scene works as intended now, but I get this message twice: 
![case missmatch](https://github.com/ramokz/phantom-camera/assets/166177985/8af40461-8a77-4a2e-bff8-5982b9cbcb62)
and I don't know if I should be worried since everything works as intended.
I'm new to programming, so I don't get a lot of the basic things, but I understand the issue is in the files' names, although it doesn't seem to affect the scene I'm working on right now.

**Comment by ramokz** [comment, marked as answer]
It's an annoyingly common issue when files have been renamed. It has something to do with Godot storing a reference to the original file name in a hidden project file, but not recognising the file name has changed. If you're not encountering any errors, and everything works still, then it should be safe to ignore.

Some have removed the output messages by either:
- Relaunching the project multiple times — as dumb as it sounds
- Deleting the `.godot` directory and let it regenerate when opening the project again
  - **Note:** this will revert any custom editor settings you've applied to your project
- Deleting the addon and reinstalling it

**Reply by WyvernZ22** [reply]
I'll keep an eye on the messages while I keep working on the project.  It does seem a bit silly, but that's the world of Godot lol.  Thank you!
